ckan-dev/2.7/Dockerfile

```diff
@@ -1,19 +1,25 @@
-FROM openknowledge/ckan-base:2.7
+# FROM openknowledge/ckan-base:2.7
+FROM alphagov/ckan-base:2.7
```

Comment:
Does this mean we need to push `ckan-base` to our Dockerhub profile?

Reply:
For now, we have to rebuild everything, as changes to the base happen fairly frequently, when things stabilise I could push something up to make the process a bit faster

Reply:
Ah that makes sense. I was wondering if we were forced to use it because of the `alphagov/` prefix, but I see it can take it from a locally built image.